src/tools.ts

```diff
@@ -116,6 +117,16 @@ export default class Tools {
         new Notice("Advanced URI copied to your clipboard");
     }
 
+    async copyMarkdownLink(parameters: Parameters) {
```

Comment:
the `markdownTitle` should not be added to `Parameters`,because `Parameters` should only be used for uri input or output. Add it as another argument.